src/shim/hamtv0/hamt.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::error::Error;
+use super::hash_algorithm::{HashAlgorithm, Sha256};
+use super::node::Node;
+use cid::Cid;
+use forest_hash_utils::{BytesKey, Hash};
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::tuple::*;
+use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
+use fvm_ipld_encoding::CborStore;
+use serde::{de::DeserializeOwned, Serialize, Serializer};
+use std::borrow::Borrow;
+use std::marker::PhantomData;
+
+/// State of all actor implementations.
+#[derive(PartialEq, Eq, Clone, Debug, Serialize_tuple, Deserialize_tuple)]
+pub struct ActorState {
```

Comment:
Why is this here? It is not used anywhere. It can be deleted.

Reply:
Removed